.github/workflows/build.yml

```diff
@@ -62,24 +62,19 @@ jobs:
         toolchain: stable
         override: true
     - name: Run tests
-      run: cargo test --verbose --features datafusion-ext
+      run: cargo test --verbose --features datafusion-ext azure
```

Comment:
@houqp I moved azure back into test section since I figured there's no much sense to spawn 3 more jobs just to test one additional method. Btw, since it's  `cargo test` without specifying particular test case, we basically run every test again plus azure otherwise